worker/include/RTC/RTCP/CompoundPacket.h

```diff
@@ -12,21 +12,17 @@ namespace RTC { namespace RTCP
 	class CompoundPacket
 	{
 	public:
-		~CompoundPacket();
 
-		uint8_t* GetRaw();
 		size_t GetSize();
```

Comment:
Does `GetSize()` return the size in bytes of the raw compound packet? If so, could it be `GetLength()` for consistency with `RtpPacket` and others?